test/blackbox-tests/test-cases/melange/copy-files-include-subdirs.t

```diff
@@ -34,29 +34,21 @@ Example using melange.emit, copy_files and include_subdirs
   > let () = Js.log file_content
   > EOF
 
+  $ output_dir=_build/default/src/app
+  $ src=$output_dir/src/main.js
+  $ asset=$output_dir/file.txt
   $ dune build @melange
-  $ node _build/default/src/app/src/main.js
+  $ dune build $asset
```

Comment:
why do we need to build this target? `file.txt` is in the `melange` alias.

Reply:
It's more direct to check that a rule exists by building it than building an alias that is supposed to include it.